apcd-cms/src/apps/submitter_renewals_listing/views.py

```diff
@@ -0,0 +1,229 @@
+from django.http import HttpResponse, HttpResponseRedirect, JsonResponse
+from django.views.generic.base import TemplateView
+from django.template import loader
+from apps.utils.apcd_database import get_registrations, get_registration_contacts, get_submitter_info, get_registration_entities, create_submitter, update_registration, update_registration_contact, update_registration_entity
+from apps.utils.apcd_groups import is_apcd_admin, is_submitter_admin
+from apps.utils.utils import table_filter
+from apps.components.paginator.paginator import paginator
+import logging
+import json
+from dateutil import parser
+
+logger = logging.getLogger(__name__)
+
+
+class SubmittersTable(TemplateView):
```

Comment:
Note for later: I think this class can be an instance of the registrations table class, and then super over the relevant methods (get, dispatch) and edit some of the context (e.g. the `pagination_url_namespaces`)

Reply:
Great point! Done.